scripts/basic-workflow-tests.sh

```diff
@@ -108,14 +108,27 @@ test_chat() {
 test_taxonomy() {
     task Update the taxonomy
 
-    test -d taxonomy || git clone https://github.com/instructlab/taxonomy || true
+    TESTNUM=$1
+    if [ "$TESTNUM" -ne 1 ] && [ "$TESTNUM" -ne 2 ] && [ "$TESTNUM" -ne 3 ]; then
+        echo "Invalid test number: $TESTNUM"
+        exit 1
+    fi
 
-    step Make new taxonomy
-    mkdir -p taxonomy/knowledge/sports/overview/softball
+    test -d taxonomy || git clone https://github.com/instructlab/taxonomy || true
```

Comment:
Why force success on this? What if the cloning fail for a good reason? Network down or GitHub unavailable? It's probably better to catch it here.

Reply:
not new code, but i agree that `|| true` doesn't make sense